src/index.ts

```diff
@@ -16,14 +16,14 @@
 
 'use strict';
 
-const Cli = require('./cli');
+import Cli = require('./cli');
 
-module.exports = () => {
+module.exports = (): void => {
   const cli = new Cli();
   const args = process.argv.slice(2);
   // TODO(peconn): Tighten up the `any` below once CLI is typed.
```

Comment:
You can remove the TODO now (and thanks for doing it for me!).

Reply:
That's the upside of having a working linter. :)